verbose_debugging_reports.md

```diff
@@ -290,4 +298,4 @@ The trigger was rejected due to an internal error.
 [trigger algorithm]: https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#trigger-attribution-algorithm
 [trigger-state cardinality limit]: https://wicg.github.io/attribution-reporting-api/#max-trigger-state-cardinality
 [unattributed reporting origin limit]: https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#reporting-origin-limits
-[verbose debugging reports]: https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#verbose-debugging-reports
\ No newline at end of file
+[verbose debugging reports]: https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#verbose-debugging-reports
```

Comment:
What changed on this line?

Reply:
I don't know. It doesn't show in diff locally.